src/main/java/seedu/address/model/event/IsolatedEventList.java

```diff
@@ -98,6 +99,49 @@ public void edit(IsolatedEvent originalEvent, IsolatedEvent editedEvent) {
         isolatedEvents.remove(originalEvent);
         isolatedEvents.add(editedEvent);
     }
+
+    /**
+     * This function cross-check with the recurring event list to check for any conflicts
+     * @param isolatedEvent is the event to be added
+     * @param recurringEventList is the event list to be checked with
+     * @throws EventConflictException if there is a conflicted event
+     */
+    public static void listConflictedEventWithRecurring(
```

Comment:
This is good as it is a better OOP as IsolatedEventList should be the one checking for the conflicts instead of the Add commands for events